docs/source/cuda/overview.rst

```diff
@@ -57,6 +57,31 @@ If you are not using Conda or if you want to use a different version of CUDA
 toolkit, the following describe how Numba searches for a CUDA toolkit
 installation.
 
+.. _cuda-bindings:
+
+CUDA Bindings
+~~~~~~~~~~~~~
+
+Numba supports interacting with the CUDA Driver API via the `NVIDIA CUDA Python
+bindings <https://nvidia.github.io/cuda-python/>`_ and its own ctypes-based
+binding. The ctypes-based binding is presently the default as Per-Thread
```

Comment:
```suggestion
binding. The ctypes-based binding is presently the default as the Per-Thread
```

Reply:
Making a slightly different grammatical change here, because "Per Thread Default Streams" is not an API, so "the Per-Thread Default Streams" doesn't quite make sense.

Reply:
The proposed change looks good, thanks.